src/transformers/models/flava/modeling_flava.py

```diff
@@ -1375,6 +1375,8 @@ def forward(
         >>> image_embeddings = outputs.image_embeddings
         >>> text_embeddings = outputs.text_embeddings
         >>> multimodal_embeddings = outputs.multimodal_embeddings
+        >>> print(image_embeddings.shape, text_embeddings.shape, multimodal_embeddings.shape)
+        torch.Size([1, 197, 768]) torch.Size([1, 7, 768]) torch.Size([1, 205, 768])
```

Comment:
I am not sure this works ..(?)

Reply:
I ran the test with `pytest --doctest-modules src/transformers/models/flava/modeling_flava.py::FlavaModel.forward` and it passes, let me know if you had something else in mind!

Reply:
oh, I see, it is in a print statement so it works. Still not very prettry.

Usually we do sth like

```
>>> outputs.image_embeddings
torch.Size([1, 197, 768])

>>> outputs.text_embeddings
torch.Size([1, 7, 768])
```

Reply:
I see ok ok, happy to change that as well